src/js/components/RangeSelector/stories/Simple.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Box } from 'grommet';
+import { RangeSelector } from '../RangeSelector';
+
+export const Simple = () => {
+  const [range, setRange] = useState();
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box pad="xlarge">
+      <RangeSelector
```

Comment:
I'm no longer able to adjust the RangeSelector value in this story

Reply:
fixed